### PR TITLE
Atom feed

### PR DESCRIPTION
**Pull request by Colin-Lane**
Initial Atom
- atom has a logo field, didn't know where to get the logo from.
- also atom can add contributors as an xml field (didn't add that to keep it the same as RSS)
- still need licensing in the rights field.

@andrewconner @stkem

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/common/seo/AtomCommander.scala [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+import com.keepit.model._
+import com.keepit.rover.RoverServiceClient
+import org.joda.time.DateTime
+import scala.concurrent.ExecutionContext.Implicits.global
+
+import scala.concurrent.Future
+import scala.xml.{ Elem, Text }
+
+/**
+ * Created by colinlane on 5/28/15.
```
Comment (stkem) [inline kifi-backend/modules/shoebox/app/com/keepit/common/seo/AtomCommander.scala]: We don't usually do this. That is what `git blame` is for :-)

File: kifi-backend/modules/shoebox/app/com/keepit/common/seo/AtomCommander.scala [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+import com.keepit.commanders.{ KeepImageCommander, ScaleImageRequest, LibraryImageCommander }
+import com.keepit.common.db.slick.Database
+import com.keepit.common.logging.Logging
+import com.keepit.common.store.{ ImageSize, S3ImageConfig }
+import com.keepit.common.time._
+import com.keepit.inject.FortyTwoConfig
+import com.keepit.model._
+import com.keepit.rover.RoverServiceClient
+import org.joda.time.DateTime
+import scala.concurrent.ExecutionContext.Implicits.global
```
Comment (stkem) [inline kifi-backend/modules/shoebox/app/com/keepit/common/seo/AtomCommander.scala]: Please use the play execution context `play.api.libs.concurrent.Execution.Implicits._`, not the default scala one. Or, even better, inject an execution context if that is possible here.

Comment (stkem) [inline kifi-backend/modules/shoebox/app/com/keepit/common/seo/AtomCommander.scala]: The scala execution context is not very configurable, whereas the play one is configured by us in the application config.

File: kifi-backend/modules/shoebox/test/com/keepit/common/seo/AtomCommanderTest.scala [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+import com.keepit.test.ShoeboxTestInjector
+import org.joda.time.DateTime
+import org.specs2.mutable.Specification
+
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+import scala.xml.NodeSeq
+
+/**
+ * Created by colinlane on 5/28/15.
```
Comment (stkem) [inline kifi-backend/modules/shoebox/test/com/keepit/common/seo/AtomCommanderTest.scala]: Ditto, my comment on the commander

**Comment by stkem** [comment]
made a few comments. lgtm otherwise. Don't worry about logo etc. to start with. We'll add those on second pass.
@andrewconner do you know the status on that license?

File: kifi-backend/modules/shoebox/app/com/keepit/common/seo/AtomCommander.scala (hunk 1 above)
Comment (Colin-Lane) [inline kifi-backend/modules/shoebox/app/com/keepit/common/seo/AtomCommander.scala]: Haha, that was autocreated by Intellij upon class creation. I'll mess with the class creation templates.

**Pull request merged**